perfkitbenchmarker/benchmarks/speccpu2006_benchmark.py

```diff
@@ -219,6 +221,8 @@ def Run(benchmark_spec):
   vm = vms[0]
   logging.info('SpecCPU2006 running on %s', vm)
   num_cpus = vm.num_cpus
+  vm.RemoteCommand("cp {0}/config/Example-simple.cfg "
```

Comment:
What's going on here? It shouldn't be necessary to modify benchmark code to add support for a new provider. If there's a bug in the original install process, we should track that separately, but I suspect this may be a leftover experiment.